tests/run-tests.sh

```diff
@@ -67,10 +67,21 @@ function initiate_test_suite {
 }
 
 function set_test_matrix {
+    gpu_platform="NVIDIA"
+    if [[ "$#" -gt 0 ]]; then
+       INPUT=$1
```

Comment:
Should this have GPU in the name so that we know what it's supposed to be? GPU_INPUT or GPU_PARAM ?

Reply:
changed the variable to `GPU_PLAT`